packages/scaffolding-cli/templates/deploy/k8s/app/base/deployment.yml

```diff
@@ -51,7 +51,15 @@ spec:
                 secretKeyRef:
                   name: appinsights
                   key: INSTRUMENTATIONKEY
-
+            
+            - name: NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN
+              value: '9L3ML2_zyQ2FrnFvuXeWES2wPWVqEnQj0Lx14UtAx4o'
+            - name: NEXT_PUBLIC_CONTENTFUL_PREVIEW_ACCESS_TOKEN
+              value: '4JrZBpD7-sDb_qjD4KB3Z34eNtCR1QqgvnCYlKqZ4Wg'
+            - name: NEXT_PUBLIC_CONTENTFUL_PREVIEW_SECRET
+              value: 'Stacks'
+            - name: NEXT_PUBLIC_CONTENTFUL_SPACE_ID
+              value: 'veka2or2977j'
```

Comment:
Are these public tokens?

Reply:
@estherthetesteramido I will remove these.

Reply:
@dnitsch told they are fine to stay there?!

Reply:
Can we not declare these in the library and pull them in as env variables?

Reply:
I thought we are not using this file? I aam happy to remove it